packages/core/src/components/Dropdown/__tests__/__snapshots__/Dropdown.snapshot.test.js.snap

```diff
@@ -880,386 +880,7 @@ exports[`Dropdown renders correctly snapshot driver should use async if set 1`]
-                  >
-                    8
-                  </div>
-                </div>
-              </div>
-            </div>
-          </div>
-          <div
-            style="position: absolute; left: 0px; top: 284px; height: 35px; width: 100%;"
-          >
-            <div>
-              <div
-                aria-selected="false"
-                class="optionReset css-6ztf41-option"
-                id="react-select-29-option-8"
-                role="option"
-                tabindex="-1"
-              >
-                <div
-                  class="valueContainer"
-                  data-testid="dropdown-option-content"
-                >
-                  <div
-                    class="typography inherit start singleLineEllipsis text text2Normal"
-                    data-testid="text"
-                    style="--text-clamp-lines: 1;"
-                  >
-                    9
-                  </div>
-                </div>
-              </div>
-            </div>
-          </div>
-          <div
-            style="position: absolute; left: 0px; top: 319px; height: 35px; width: 100%;"
-          >
-            <div>
-              <div
-                aria-selected="false"
-                class="optionReset css-6ztf41-option"
-                id="react-select-29-option-9"
-                role="option"
-                tabindex="-1"
-              >
-                <div
-                  class="valueContainer"
-                  data-testid="dropdown-option-content"
-                >
-                  <div
-                    class="typography inherit start singleLineEllipsis text text2Normal"
-                    data-testid="text"
-                    style="--text-clamp-lines: 1;"
-                  >
-                    10
-                  </div>
-                </div>
-              </div>
-            </div>
-          </div>
-          <div
-            style="position: absolute; left: 0px; top: 354px; height: 35px; width: 100%;"
-          >
-            <div>
-              <div
-                aria-selected="false"
-                class="optionReset css-6ztf41-option"
-                id="react-select-29-option-10"
-                role="option"
-                tabindex="-1"
-              >
-                <div
-                  class="valueContainer"
-                  data-testid="dropdown-option-content"
-                >
-                  <div
-                    class="typography inherit start singleLineEllipsis text text2Normal"
-                    data-testid="text"
-                    style="--text-clamp-lines: 1;"
-                  >
-                    11
-                  </div>
-                </div>
-              </div>
-            </div>
-          </div>
-        </div>
-      </div>
-    </div>
-  </div>
-</DocumentFragment>
-`;
+exports[`Dropdown renders correctly snapshot driver should use virtualization if set 1`] = `<DocumentFragment />`;
```

Comment:
is it really a desired change? though we don't run on a real browser here, the test should mimic running on a browser

Reply:
Fixed